### PR TITLE
improve reset settings

### PR DESCRIPTION
**Pull request by ZappoMan**
- changes the name of the function to be more clear about what it does
- fixes bug that would return "previousSessionCrashed" even when it hadn't but the user had asked to reset settings

### CODE REVIEW

**Comment by ZappoMan** [comment]
Testing Notes:
- Smoke test
- Check the Developer > Ask To Reset Settings option, and verify that on next startup you are asked if you want to reset your settings [**NOTE: ask Chris or Brad to check the "metaverse" database to see that the "previousSessionCrashed" is reported as FALSE for this case**]
- Verify that after a crash you are still asked to reset your settings.  [**NOTE: ask Chris or Brad to check the "metaverse" database to see that the "previousSessionCrashed" is reported as TRUE for this case**]

**Comment by zzmp** [comment]
Amend the comment on Application.cpp L708 too.

File: interface/src/CrashHandler.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ bool CrashHandler::checkForAndHandleCrash() {
                 handleCrash(action);
             }
         }
-        return true;
+        return wasLikelyCrash;
```
Comment (zzmp) [inline interface/src/CrashHandler.cpp]: Can you delete this line and just have on return statement, `return wasLikelyCrash`?

Comment (ZappoMan) [inline interface/src/CrashHandler.cpp]: ok

**Comment by hifi-gustavo** [comment]
Build console output for commit: 67af3917bc6d358917500bc35f96c8e8a2df14c1 available at: https://gist.github.com/86fae944c9a8122fb4bba6fe3583e02d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7659.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7659.dmg

**Comment by ZappoMan** [comment]
I believe that comment remains accurate as is...

```
    // sessionRunTime will be reset soon by loadSettings. Grab it now to get previous session value.
    // The value will be 0 if the user blew away settings this session, which is both a feature and a bug.
    UserActivityLogger::getInstance().launch(applicationVersion(), _previousSessionCrashed, sessionRunTime.get());
```

The "value" in this case I believe refers to the `sessionRunTime` -- the behavior for this remains the same with my PRs.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9c437c082469e5b88f48f7601e202e246c776716 available at: https://gist.github.com/86fae944c9a8122fb4bba6fe3583e02d<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7659.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Alpha-PR7659.dmg

**Comment by cozza13** [comment]
tested and looked to metaverse DB to ensure change. QA pass

**Pull request merged**